### DISCUSSION TITLE
Approve Actions for future commits by new contributor.

### DISCUSSION THREAD

**Discussion by domdfcoding** [community/community#35064] [Actions]
I've got a PR from a new contributor (let's call him Bob). We've had a bit of back and forth regarding changes, but every time Bob pushes new changes I have to click the button to give approval for Actions to run. By this point I trust that Bob's a stand-up guy and isn't going to do anything naughty (as much as I'd trust an existing contributor, anyway), so I'd like to approve Actions for all future commits on this PR. 

However, there doesn't seem to be such an option, just the ability to disregard approval entirely for all PRs from new contributors. Am I missing something?

**Comment by airtower-luna** [comment]
There's no such option for the *first* PR by a person, but you can say that after that first PR has been merged actions for further ones by the same person should run automatically (in the repository settings under "Actions"):

![grafik](https://user-images.githubusercontent.com/5814919/193806143-ff20b00e-919e-4f50-bd35-de1949964da4.png)
